src/orion/benchmark/task/profet/profet_task.py

```diff
@@ -0,0 +1,556 @@
+    ),
+    svm=dict(
+        C="loguniform(np.exp(-10), np.exp(10))",
+        gamma="loguniform(np.exp(-10), np.exp(10))",
+    ),
+    fcnet=dict(
+        learning_rate="loguniform(1e-6, 1e-1)",
+        batch_size="loguniform(8, 128, discrete=True)",
+        units_layer1="loguniform(16, 512, discrete=True)",
+        units_layer2="loguniform(16, 512, discrete=True)",
+        dropout_rate_l1="uniform(0, 0.99)",
+        dropout_rate_l2="uniform(0, 0.99)",
+    ),
+    xgboost=dict(
+        learning_rate="loguniform(1e-6, 1e-1)",
+        gamma="uniform(0, 2, discrete=False)",
+        l1_regularization="loguniform(1e-5, 1e3)",
+        l2_regularization="loguniform(1e-5, 1e3)",
+        nb_estimators="uniform(10, 500, discrete=True)",
+        subsampling="uniform(0.1, 1)",
+        max_depth="uniform(1, 15, discrete=True)",
+        min_child_weight="uniform(0, 20, discrete=True)",
+    ),
+)
+
+get_architecture: Dict[str, Callable[[int], nn.Module]] = dict(
+    forrester=get_architecture_forrester,
+    svm=functools.partial(get_default_architecture, classification=True),
+    fcnet=functools.partial(get_default_architecture, classification=True),
+    xgboost=get_default_architecture,
+)
+
+
+names: Dict[str, str] = dict(
+    forrester="data_sobol_forrester.json",
+    svm="data_sobol_svm.json",
+    fcnet="data_sobol_fcnet.json",
+    xgboost="data_sobol_xgboost.json",
+)
+
+hidden_space: Dict[str, int] = dict(forrester=2, fcnet=5, svm=5, xgboost=5)
+
+normalize_targets: Dict[str, bool] = dict(
+    forrester=True, fcnet=False, svm=False, xgboost=True
+)
+
+log_cost: Dict[str, bool] = dict(forrester=False, fcnet=True, svm=True, xgboost=True)
+
+log_target: Dict[str, bool] = dict(
+    forrester=False, fcnet=False, svm=False, xgboost=True
+)
+
+
+def load_data(
+    input_path: Union[str, Path], benchmark: str
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    file = Path(input_path) / "profet_data" / names[benchmark]
+    if not file.exists():
+        logger.info(f"File {file} doesn't exist, attempting to download data.")
+        download_data(input_path)
+        logger.info(f"Download finished.")
+        assert file.exists(), input_path
+    res = json.load(open(file, "r"))
+    X, Y, C = np.array(res["X"]), np.array(res["Y"]), np.array(res["C"])
+    if len(X.shape) == 1:
+        X = X[:, None]
+
+    return X, Y, C
+
+
+def normalize_Y(
+    Y: np.ndarray, indexD: np.ndarray
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    max_idx = np.max(indexD)
+    Y_mean = np.zeros(max_idx + 1)
+    Y_std = np.zeros(max_idx + 1)
+    for i in range(max_idx + 1):
+        Y_mean[i] = Y[indexD == i].mean()
+        Y_std[i] = Y[indexD == i].std() + 1e-8
+        Y[indexD == i] = (Y[indexD == i] - Y_mean[i]) / Y_std[i]
+    return Y, Y_mean[:, None], Y_std[:, None]
+
+
+def get_features(
+    X: np.ndarray, Y: np.ndarray, C: np.ndarray, hidden_space: int
+) -> Tuple[np.ndarray, np.ndarray]:
+    n_inducing_lvm = 50
```

Comment:
May be worth making these configurable. Especially to reduce running costs for tests (smaller max_iters).